HealthCheck/Tools/class-health-check-beta-features.php

```diff
@@ -66,5 +59,3 @@ public function tab_content() {
 	}
 
 }
-
-new Health_Check_Beta_Features();
```

Comment:
This is now instantiated during `init_tools()` method which is hooked in the `init` action with priority 1.